command/polly_tts.js

```diff
@@ -7,19 +7,23 @@ var glob = require('glob');
 var polly_tts = {
   getPollyChunk: function(text, filenameIndex, audio_file) {
     return new Promise(function(resolve, reject) {
+      let rate = process.env.PROSODY_RATE || 'medium';
+      var ssmlText =
+        '<speak><prosody rate="' + rate + '">' + text + '</prosody></speak>';
+
       let params = {
-        Text: text,
+        Text: ssmlText,
         OutputFormat: 'mp3',
         SampleRate: '16000',
-        VoiceId: process.env.POLLY_VOICE || 'Kimberly'
+        VoiceId: process.env.POLLY_VOICE || 'Kimberly',
```

Comment:
Update to Salli so we can close out #72 too?

Reply:
done

Reply:
BTW, I think Salli is much worse than Kimberly